common/src/main/java/tachyon/security/authentication/PlainSaslServer.java

```diff
@@ -213,7 +213,7 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
   /**
    * An instance of this class represents a client user connecting to Tachyon service.
    *
-   * It is maintained in a ThreadLocal variable based on the Thrift RPC mechanism.
+   * It is maintained in a {@link ThreadLocal} variable based on the Thrift RPC mechanism.
```

Comment:
For standard library classes like `ThreadLocal`, would it make sense to only `@link` them the first time they are mentioned in a file? There's no benefit to being able to rename them with an IDE.